README.md

```diff
@@ -49,7 +46,7 @@ If there are issues or crashes with Draconic Evolution, please report them to [h
 
 ## Credits
 Original pack by [Exaxxion](https://github.com/Exaxxion).  
-Certain new quests from [GregTech Community Pack](https://github.com/GregTechCEu/GregTech-Community-Pack).  
+Certain new quests from [GregTech Community Pack](https://github.com/GregTechCEu/GregTech-Community-Pack).
```

Comment:
Please add back these spaces.

Reply:
I'm pretty sure there's a way to revert specific lines, will do